src/components/ActionTooltip.vue

```diff
@@ -152,7 +152,7 @@ export default {
       })
     },
     /**
-     * The primary cost type; 0 for consumes mp, > 0 for something else
+     * The primary cost type; 0 is no cost, 2 is HP, 3 is MP, 5 is TP, 7 is GP, 8 is CP
```

Comment:
Thank you for updating this comment.
Since we now know the types, we should probably update [this line](https://github.com/nonowazu/vue-xivtooltips/blob/master/src/components/ActionTooltip.vue#L220) to be a computed property a-la something like

```js
computed: {
    // resourceMapping defined somewhere else
    resourceType () { return resourceMapping[this.PrimaryCostType] },
    ....
}
```

and then in the template

```html
                    <div class="xivtooltip-c xivtooltip-text">
                        {{ resourceType }} Cost
                    </div>
```